charts/traction/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: traction
 description: The Traction service allows organizations to verify, hold, and issue verifiable credentials. The Traction Tenant UI allows tenants to manage their agent.
 type: application
-version: 0.3.5
+version: 0.3.6
```

Comment:
I think `0.3.5` is actually the next version (it was already updated).

Reply:
Updated.